articles/stream-analytics/stream-analytics-documentdb-output.md

```diff
@@ -32,7 +32,12 @@ Stream Analytics integration with Azure Cosmos DB allows you to insert or update
 
 Stream Analytics uses an optimistic upsert approach, where updates are only done when insert fails with a Document ID conflict. This update is performed as a PATCH, so it enables partial updates to the document, that is, addition of new properties or replacing an existing property is performed incrementally. However, changes in the values of array properties in your JSON document result in the entire array getting overwritten, that is, the array isn't merged.
 
-If the incoming JSON document already has an ID field in it, that field is used as the Document ID column, and any subsequent writes are handled as such, determined by the Document ID setting in the output settings, as specified below. If you receive documents without a unique value in ID and want to save all those, then the way to solve this is to rename the ID field in your query. THen if you do want to overwrite when a new file come in with the same ID, just specify the Document ID setting to 'id' and it will upsert based on that field.
+If the incoming JSON document has an existing ID field, that field is automatically used as the Document ID column in Cosmos DB and any subsequent writes are handled as such, leading to one of these situations:
+- unique ID's: insert
+- duplicate ID's and 'Document ID' set to 'ID': upsert
+- duplicate ID's and 'Document ID' not set: error
```

Comment:
Please change to:

- Unique IDs lead to insert.
- Duplicate IDs and 'Document ID' set to 'ID' leads to upsert.
- Duplicate IDs and 'Document ID' not set leads to error.